sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_multiple_node/language-modeling-multi-gpu-multi-node.ipynb

```diff
@@ -2,15 +2,15 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        entry_point=f"run_{LANGUAGE_MODELING_LOSS}.py",  # Wrapper around training script that enables multi node training
</pre><p>Remove stale comment about wrapper script.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>